utils/management/commands/add_oidc_client_to_api_scope.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from oidc_provider.models import Client
+
+from oidc_apis.models import ApiScope
+
+
+class Command(BaseCommand):
+    help = "Add New Client OpenID Connect API Scope"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-c", "--client_id", type=str, help="Client ID", required=True
+        )
+        parser.add_argument(
+            "-asi",
+            "--api_scope_identifier",
+            type=str,
+            help="API Scope Identifier",
+            required=True,
+        )
+
+    @transaction.atomic
```

Comment:
This seems unnecessary.

Reply:
Removed. Originates from other oidc-client related actions.